projects/plugins/protect/src/js/components/footer/index.jsx

```diff
@@ -21,51 +27,78 @@ const ProductPromotion = () => {
+	// TODO: Update with new paid Protect product
 	const { securityBundle } = useProtectData();
 	const { hasRequiredPlan } = securityBundle;
 
+	// TODO: Update button links with appropriate Cloud links
 	if ( hasRequiredPlan ) {
-		const getStartedUrl = getRedirectUrl( 'protect-footer-get-started-scan' );
-
 		return (
 			<div className={ styles[ 'product-section' ] }>
-				<IconsCard products={ [ 'backup', 'scan', 'anti-spam' ] } />
-				<Title>
-					{ __( 'Learn how Jetpack Scan increases your site protection', 'jetpack-protect' ) }
-				</Title>
+				<Title>{ __( 'Get access to our Cloud', 'jetpack-protect' ) }</Title>
 				<Text mb={ 3 }>
 					{ __(
-						'With your Jetpack Security bundle you have access to Jetpack Scan. Automatically scan your site from the Cloud, get email notifications and perform one-click fixes.',
+						'With your Protect upgrade, you have free access to scan your site on our Cloud, so you can be aware and fix your threats even if your site goes down. ',
 						'jetpack-protect'
 					) }
+					<Button variant="link" weight="regular" href="#">
+						{ __( 'Learn more', 'jetpack-protect' ) }
+					</Button>
 				</Text>
 
-				<Button variant="link" isExternalLink={ true } weight="regular" href={ getStartedUrl }>
-					{ __( 'Get Started', 'jetpack-protect' ) }
+				<Button variant="secondary" weight="regular" href="#">
+					{ __( 'Go to Cloud', 'jetpack-protect' ) }
 				</Button>
 			</div>
 		);
 	}
 
 	return (
 		<div className={ styles[ 'product-section' ] }>
-			<IconsCard products={ [ 'scan' ] } />
-			<Title>{ __( 'Comprehensive Site Security', 'jetpack-protect' ) }</Title>
+			<Title>{ __( 'Advanced scan results', 'jetpack-protect' ) }</Title>
 			<Text mb={ 3 }>
 				{ __(
-					'Jetpack Security offers advanced scan tools, including one-click fixes for most threats and malware scanning. Plus, with this bundle you also get real-time cloud backups and spam protection.',
+					'Upgrade Jetpack Protect to get advanced scan tools, including one-click fixes for most threats and malware scanning.',
 					'jetpack-protect'
 				) }
 			</Text>
 
-			<Button variant="secondary" onClick={ getSecurityBundle } isLoading={ hasCheckoutStarted }>
-				{ __( 'Get Jetpack Security', 'jetpack-protect' ) }
-			</Button>
+			<ContextualUpgradeTrigger
+				description={ __(
+					'Looking for advanced scan results and one-click fixes?',
+					'jetpack-protect'
+				) }
+				cta={ __( 'Upgrade Jetpack Protect now', 'jetpack-protect' ) }
+				onClick={ getSecurityBundle }
+			/>
 		</div>
 	);
 };
 
 const FooterInfo = () => {
+	// TODO: Update with new paid Protect product
+	const { securityBundle } = useProtectData();
+	const { hasRequiredPlan } = securityBundle;
+
+	// TODO: Update button link with learn more link for the paid Protect product
+	if ( hasRequiredPlan ) {
+		return (
+			<div className={ styles[ 'info-section' ] }>
+				<Title>{ __( 'Line-by-line scanning', 'jetpack-protect' ) }</Title>
+				<Text mb={ 3 }>
```

Comment:
```suggestion
				<Text mb={ 2 }>
```
Minor suggestion - This will reduce the spacing between the text and "Learn more" button to `16px` to match design :+1: